aruco_message[] detected_arucos
